### PR TITLE
[feature1] First feature was implemented

### PR DESCRIPTION
**Pull request by div41758**
Following changes were made:

- new-feature.txt file was added.
- feature1 folder was created.

### CODE REVIEW

File: feature1/one-more-file.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+This file is also added in the feature1 branch under feature folder of root project directory
```
Comment (div4321) [inline feature1/one-more-file.txt:1]: this looks perfectly fine and works as expected

**Comment by div4321** [comment]
thanks for your valuable feedback

File: feature1/one-more-file.txt (hunk 1 above)
Comment (div41758) [reply feature1/one-more-file.txt:1]: thanks for your valuable feedback!!

**Review by div4321 — APPROVED** [review]
Well Done!!

**Pull request merged**